packages/serverless-workflow-combined-editor/src/editor/helpers/PaintNodes.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2023 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Node } from "@kie-tools/serverless-workflow-diagram-editor-envelope/dist/api/StunnerEditorEnvelopeAPI";
+
+const paintCompletedNode = async (node: Node, color: string) => {
```

Comment:
Could you please remove all the references to node status (like "completed"), to keep it generic? Also, to keep everywhere the same name, please change all "paint" to "color".

For example:
```suggestion
const colorNode = async (node: Node, color: string) => {
```

Reply:
changed all reference from paint to color